myStyle.css

```diff
@@ -0,0 +1,38 @@
+  }
+div p {
+    color: navy;
+    margin-left: 20px;
+  }
+
+#ID {
+    background-color: red;
+  }
+
+.nice {
+    font-family: sans-serif;
+  }
+
+[target=_blank] {
+    background-color: yellow;
+  }
+
+p[target=_hi] {
+    background-color: rgb(212, 0, 255);
```

Comment:
it is clearer to the reader to use named colors instead of rgb values
https://www.w3schools.com/colors/colors_names.asp